src/components/dashboard/sidebar/SubsidiesSummary.jsx

```diff
@@ -77,15 +76,15 @@ const SubsidiesSummary = ({
 
   return (
     <>
-      {hasAssignedSubsidyOrRequests && (
+      {hasAssignedSubscriptionOrCodeOrOffer && (
         // TODO: Design debt, don't have cards in a card
         <SidebarCard
           cardSectionClassNames={
-            `${programProgressPage && 'border-remove'}`
+            `border-remove ${programProgressPage}`
           }
           cardClassNames={
             `mb-5 ${programProgressPage ? 'col-8 border-remove'
-              : 'border-primary border-brand-primary catalog-access-card'}`
+              : 'border-primary'}`
```

Comment:
I think we'll want to nix the `border-primary` class name as well, since it appears we want the top-most `Card` to have the standard gray border (no primary color anymore).